tests/layer_tests/tensorflow_tests/test_tf_Bincount.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import platform
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+class TestBincount(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'x:0' in inputs_info, "Test error: inputs_info must contain `x`"
+        x_shape = inputs_info['x:0']
+
+        inputs_data = {}
+        inputs_data['x:0'] = np.random.randint(0, np.max(x_shape), x_shape)
+
+        if 'w:0' in inputs_info:
+            w_shape = inputs_info['w:0']
+            inputs_data['w:0'] = np.random.rand(*w_shape).astype(self.weights_type)
+
+        return inputs_data
+
+    def create_bincount_net(self, input_shape, size, weights, weights_type):
+        tf.compat.v1.reset_default_graph()
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            x = tf.compat.v1.placeholder(np.int32, input_shape, 'x')
+            s = tf.constant(size)
+            self.weights_type = weights_type
+            if weights is not None:
+              w = tf.compat.v1.placeholder(weights_type, input_shape, 'w')
+            else:
+              w = tf.constant([], dtype=weights_type)
+
+            tf.raw_ops.Bincount(arr=x, size=s, weights=w)
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        return tf_net, None
+
+    def gen_data(weights, weights_type):
+      input_shape = tuple(np.random.randint(1, 10, size=np.random.randint(1, 5)))
+      size = np.random.randint(1, np.max(input_shape) + 1)
+      return dict(input_shape=input_shape, size=size, weights=weights, weights_type=weights_type)
+  
+    test_data = [
+        # with no weights
+        gen_data(None, np.float32),
+        gen_data(None, np.float64),
+        gen_data(None, np.int32),
+        gen_data(None, np.int64),
+
+        # with weights
+        gen_data(True, np.float32),
+        gen_data(True, np.float64),
+        gen_data(True, np.int32),
+        gen_data(True, np.int64),
+    ]
```

Comment:
please set concrete input shapes without its generatation. We only generate input data above and we usualy have input shapes to be determined

Reply:
Let us check input_shape: 0-rank, 1-rank, 2-rank and 3-rank. For example, [], [2], [1, 3], [3, 1, 4]. Let us have not big shapes just to save computing resources. For tests it will be sufficient

Reply:
got it, i see

Reply:
another question is, 
when I `unset TEST_DEVICE` or forgot to set to `CPU`, seems all test cases will fail on GPU, it says

<img width="1602" alt="image" src="https://github.com/openvinotoolkit/openvino/assets/14276970/4145c229-d8df-4b91-86f8-7a2cf45ecc88">

Is it OK to ignore these error

my system info: 
<img width="521" alt="image" src="https://github.com/openvinotoolkit/openvino/assets/14276970/2991af49-b609-4ca5-ac95-4c8e44c9981c">